libs/sdk-core/src/bridge_generated.rs

```diff
@@ -1071,6 +1071,7 @@ impl support::IntoDart for InputType {
                 vec![0.into_dart(), address.into_into_dart().into_dart()]
             }
             Self::Bolt11 { invoice } => vec![1.into_dart(), invoice.into_into_dart().into_dart()],
+            Self::Bolt12Offer { offer } => todo!(),
```

Comment:
This needs to be generated. Please look at `sdk-flutter`and `sdk-react-native` to generate the code for these.